src/js/components/githubIssues/selectIssueModal.tsx

```diff
@@ -0,0 +1,187 @@
+          variant={isOpen === 'task' ? 'brand' : 'outline-brand'}
+          disabled={!selectedIssue}
+          label={i18n.t('Create a Task')}
+          onClick={() => onSubmit(selectedIssue, 'task')}
+        />,
+      ]}
+    >
+      <div className="slds-is-relative slds-p-around_large">
+        <p>
+          <Trans i18nKey="githubIssuesHelp">
+            Issues are items in GitHub’s bug and enhancement tracking system.
+            Select from these open Issues, and create a Task or Epic.
+          </Trans>
+        </p>
+        {issues && attachedIssues ? (
+          <form className="slds-form slds-p-top_large">
+            <div className="slds-grid slds-gutters">
+              <div className="slds-col slds-size_1-of-2">
+                <h2 className="slds-text-heading_small">
+                  {i18n.t('Available Issues')}
+                </h2>
+                <RadioGroup
+                  assistiveText={{
+                    label: i18n.t('Available Issues'),
+                    required: i18n.t('Required'),
+                  }}
+                  className="slds-form-element_stacked slds-p-left_none"
+                  name="github-issue"
+                  required
+                  onChange={changeSelection}
+                >
+                  {issues.length ? (
+                    issues.map((issue, idx) => (
+                      <div key={idx} className="slds-p-vertical_x-small">
+                        <Radio
+                          labels={{ label: `#${issue.number}: ${issue.title}` }}
+                          name="github-issue"
+                          value={issue.id}
+                          checked={selectedIssue === issue.id}
+                        />
+                        <GitHubIssueLink url={issue.html_url} />
+                      </div>
+                    ))
+                  ) : (
+                    <p>{i18n.t('No Available Issues')}</p>
+                  )}
+                </RadioGroup>
+              </div>
+              <div className="slds-col slds-size_1-of-2">
+                <h2 className="slds-text-heading_small">
+                  {i18n.t('Attached Issues')}
+                </h2>
+                <RadioGroup
+                  assistiveText={{
+                    label: i18n.t('Attached Issues'),
+                  }}
+                  className="slds-form-element_stacked slds-p-left_none"
+                  name="attached-github-issue"
+                >
+                  {attachedIssues.length ? (
+                    attachedIssues.map((issue, idx) => (
+                      <div key={idx} className="slds-p-vertical_x-small">
+                        <Radio
+                          labels={{ label: `#${issue.number}: ${issue.title}` }}
+                          name="attached-github-issue"
+                          checked
+                          value={issue.id}
+                          disabled
+                        />
```

Comment:
@oluoluoxenfree I also notice that this is missing some items from the mockup. Once an issue is attached, it looks like we want to display the name of the task/epic that it's attached to, with a link to the task/epic, and the status of the task/epic.

https://xd.adobe.com/view/bee12238-9700-471f-b104-3aa900d43f6e-ebb2/screen/98ab3852-0936-4ce1-a77a-89ba5f670d6e/